docs/mkdocs.yml

```diff
@@ -7,3 +7,7 @@
 - ['registry/spec/json.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; JSON format' ]
 - ['registry/spec/auth/token.md', 'Reference', '&nbsp;&nbsp;&nbsp;&nbsp;&blacksquare;&nbsp; Authenticate via central service' ]
 
+- ['registry/storage-drivers/azure.md', '**HIDDEN**' ]
+- ['registry/storage-drivers/filesystem.md', '**HIDDEN**' ]
+- ['registry/storage-drivers/inmemory.md', '**HIDDEN**' ]
+- ['registry/storage-drivers/s3.md', 'Reference', '**HIDDEN**' ]
```

Comment:
Yikes,  passed you bad data.  Can you change this line to :

+- ['registry/storage-drivers/s3.md', '**HIDDEN**' ]